### PR TITLE
refactor: export beacon utils to be reused by other packages

### PR DESCRIPTION
**Pull request by nazarhussain**
**Motivation**

Make the packages and complex code more re-useable. 

**Description**

We need to use `kzg` implementation and trusted setup during the blobs testing on the CLI package and sim tests.

It would be difficult to maintain if we duplicate logic like `kzg` in different packages, so we have to export such utils. 

**Limit the utils export only to important ones, which are integral for application behavior.**

**Steps to test or reproduce**

- Run all tests

### CODE REVIEW

**Review by matthewkeil — APPROVED** [review]
LGTM!! 🚀

File: packages/beacon-node/src/util/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+//  Only export the functions which are useful for external packages and difficult to reproduce or mock
+export * from "./kzg.js";
```
Comment (nflaig) [inline packages/beacon-node/src/util/index.ts:2]: Where do we need this?

Comment (nazarhussain) [reply packages/beacon-node/src/util/index.ts:2]: To create sim tests for 4844, we need trusted setup. To create and sign blobs.

**Review by nflaig — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.18.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.18.0) :tada:

**Pull request merged**